<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timecard</class>
 <widget class="QDialog" name="Timecard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maintain Timecard</string>
  </property>
  <widget class="QTextEdit" name="textEditDisplay">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUpdate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSubmit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
